eth/1-add-0x3041-to-0x2A82/README.md

```diff
@@ -68,39 +70,32 @@ you’ll need to validate the following items in the simulation:
 
 The domain hash (item 4 in the signing output above) should match the Tenderly `domainSeparator`.
 
-Example where the hash is 0xf347c…:
+For this ceremony, the hash should be 0x462891ca8f7777ee79e5fe4366d0577864fdf00769c59e345303b084bf0cf38d:
 
-![](../../images/tenderly-separator.png)
+![](./images/domain-hash.png)
 
-#### 6.2. The data hash
+#### 6.2. The message hash
 
 The data hash to sign (item 3 in the signing output) should match the
 data field when checking the signatures. Example where data is
-0x1901f3…:
+0xbd19b32a5cbf3cdb557327bd20851b9de8232ab4353f5911bac6b7d95724b700:
 
-![](../../images/tenderly-data.png)
+![](./images/message-hash.png)
 
 #### 6.3. The state changes
 
 Now click on the ‘State’ tab. Verify that the ‘Before’ and ‘After’
 values under ‘State Changes’ implements the Objective above and no
 other changes are made.
```

Comment:
This came up in the signer removal rehearsal, and will appear in any removal or adding simulation. 

```suggestion
other changes are made. Note that it will appear as if the threshold is currently `1`, this is a side effect of the state overrides which are necessary to simulate the transaction. The important thing is that the new threshold matches the updated value of `3`.
```